### PR TITLE
Bump gl-native to v10.9.0-beta.1, common to v23.1.0-beta.1.

### PR DESCRIPTION
**Pull request by pengdev**
## Features ✨ and improvements 🏁
* Add experimental line-edge-clip property
* Introduce `location-indicator-opacity` to location indicator layer that allows to control layer opacity.
* Enable asyncronous tile uploader by default to improve animation performance
* Vector tiles without symbols are not hold for fade-out animation so that less amount of vector tiles are managed at a time
* Introduce maps Version API
* Added `line-border-width` and `line-border-color` experimental line layer properties.

## Bug fixes 🐞
* Fix pixel flickering between tiles on darker styles in globe view.
* Fix incorrect shading of rounded corners in fill extrusions when ambient occlusion and/or shadow is enabled.
* Fix shadows when the light radial coordinate is zero
* Location indicator layer is occluded by terrain to align rendering behavior with other location indicator rendering types
* Symbol flickering after runtime styling caused by using outdated symbol placement on updated symbol buckets
* Allow Light intensity and Light shadow intensity to accept expressions when using setStyleLight API call
* Fix a bug in cameraForGeometry when called on a map with padding set.

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [x] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

**Review by jush — APPROVED** [review]
LGTM but pending ✅ from gl-native about `line-*` properties

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Features ✨ and improvements 🏁
 * Deprecated gesture settings `increaseRotateThresholdWhenPinchingToZoom` property. ([1632](https://github.com/mapbox/mapbox-maps-android/pull/1632))
+* Add experimental line-edge-clip property. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
+* Introduce `location-indicator-opacity` to location indicator layer that allows to control layer opacity. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
```
Comment (kiryldz) [inline CHANGELOG.md:9]: Do we need this entry? Similar will be used when our platform code will land

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Features ✨ and improvements 🏁
 * Deprecated gesture settings `increaseRotateThresholdWhenPinchingToZoom` property. ([1632](https://github.com/mapbox/mapbox-maps-android/pull/1632))
+* Add experimental line-edge-clip property. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
```
Comment (kiryldz) [inline CHANGELOG.md:8]: Perhaps remove as we will be wrapping that?

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,12 @@ Mapbox welcomes participation and contributions from everyone.
...
+* Introduce `location-indicator-opacity` to location indicator layer that allows to control layer opacity. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
+* Enable asynchronous tile uploader by default to improve animation performance. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
+* Vector tiles without symbols are not hold for fade-out animation so that less amount of vector tiles are managed at a time. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
+* Introduce maps Version API. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
```
Comment (kiryldz) [inline CHANGELOG.md:12]: I would also point how to use it if we think that deserves the changelog entry

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,12 @@ Mapbox welcomes participation and contributions from everyone.
...
+* Added `line-border-width` and `line-border-color` experimental line layer properties. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
```
Comment (kiryldz) [inline CHANGELOG.md:13]: Same as above

File: CHANGELOG.md (hunk 3 above)
Comment (pengdev) [reply CHANGELOG.md:12]: this is an internal API without doc, so I suppose we should remove it from changelog.

File: CHANGELOG.md (hunk 2 above)
Comment (pengdev) [reply CHANGELOG.md:8]: yeah, agree, will remove for now and re-introduce when we create platform wrappings.

File: CHANGELOG.md (hunk 1 above)
Comment (pengdev) [reply CHANGELOG.md:9]: yeah, will remove for now

**Review by kiryldz — APPROVED** [review]

**Pull request merged**